tests/test_language_disambiguation.py

```diff
@@ -86,4 +86,7 @@ def data_path(filename):
 def test_language_disambiguation():
     titledb = data_path('input/en_titles.db')
     semparser = EnWiktionarySemantics(titledb=titledb)
-    eq_(semparser.parse_structured_entry(ENTRY), TARGET)
+    for index, elem in enumerate(semparser.parse_structured_entry(ENTRY)):
+        for k,v in TARGET[index].items():
+            eq_(v, elem[k])
+    eq_(index, 2)
```

Comment:
I don't like relying on an index that falls out of a for loop. It would be much cleaner and clearer to assign the parsed list to a variable and assert that its length is 2.